### PR TITLE
Add resolve-function-by-brute-ratel-badger-hash.yml

### PR DESCRIPTION
**Pull request by jtothej**
Add resolve-function-by-brute-ratel-badger-hash.yml

### CODE REVIEW

File: linking/runtime-linking/resolve-function-by-brute-ratel-badger-hash.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      - https://bruteratel.com/release_notes/releases.txt
+    examples:
+      - 64ce9ab801d9bef5284b408c3373dd30ba2dc6952c0950c8049be067b5f24530:0x6DB42430
+  features:
+    - or:
+      - basic block:
+        - and:
+          - mnemonic: add
+          - instruction:
+            - mnemonic: imul
```
Comment (mr-tz) [inline linking/runtime-linking/resolve-function-by-brute-ratel-badger-hash.yml:21]: should we add `mul` here as well?

Comment (jtothej) [reply linking/runtime-linking/resolve-function-by-brute-ratel-badger-hash.yml]: Sure! Added in https://github.com/mandiant/capa-rules/pull/793/commits/20c12b55da587fc8cd72adc08ba5c78c09d6b56c

File: linking/runtime-linking/resolve-function-by-brute-ratel-badger-hash.yml [hunk 2]
```diff
@@ -17,9 +17,13 @@ rule:
       - basic block:
         - and:
           - mnemonic: add
-          - instruction:
-            - mnemonic: imul
-            - operand[2].number: 0x801
+          - or:
+            - instruction:
+              - mnemonic: imul
```
Comment (mr-tz) [inline linking/runtime-linking/resolve-function-by-brute-ratel-badger-hash.yml:22]:
```suggestion
            - instruction:
              - or:
                - mnemonic: mul
                - mnemonic: imul
```

Comment (mr-tz) [reply linking/runtime-linking/resolve-function-by-brute-ratel-badger-hash.yml:22]: this should work and you could remove the other addition

Comment (jtothej) [reply linking/runtime-linking/resolve-function-by-brute-ratel-badger-hash.yml:22]: But then I'd need to remove following `operand[2].number: 0x801` and replace it with just `number: 0x801` as `mul` does not take immediate values as operands?

Comment (mr-tz) [reply linking/runtime-linking/resolve-function-by-brute-ratel-badger-hash.yml:22]: ah, my bad, your suggestion as is, is great!

**Review by mr-tz — APPROVED** [review]

**Pull request merged**